ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/SLES.yml

```diff
@@ -23,7 +23,8 @@
 
 - name: Add Devel-Tools repository (SLES15)
   zypper_repository:
-    repo: https://download.opensuse.org/repositories/devel:/tools/15.5/devel:tools.repo
+    name: devel-tools
+    repo: 'https://download.opensuse.org/repositories/devel:/tools/15.{{ ansible_distribution_release }}/'
```

Comment:
Pedantry, would it make more sense to make ansible_distribution_release the entire 15.6?  Later on it might move to 16.x....

Reply:
@karianna , I did not change it because the when condition checks whether it is sles 15 or not.
We can either make it {{ ansible_distribution_major_version }}.{{ ansible_distribution_release }} or remove the when condition :
```
  when:
    - ansible_distribution_major_version == "15"
```